### ISSUE TITLE
JavaScript atom bot.action.clear generates blur event twice

### ISSUE THREAD

**Issue by JohnChen0** [SeleniumHQ/selenium#6557]
## Meta -
OS:  Windows, OSX, and Linux
Selenium Version:  3.14.0
Browser:  Chrome
Browser Version:  69.0.3497.100

## Expected Behavior -
After calling JavaScript atom bot.action.clear to clear a text input element, the blur event handler attached to the element is called once.

## Actual Behavior -
The blur event handler is called twice.

According to https://github.com/SeleniumHQ/selenium/blob/selenium-3.14.0/javascript/atoms/action.js#L100, the blue event handler is first called explicitly by

    bot.events.fire(element, bot.events.EventType.BLUR);

And then it is called again when the element loses focus due to

    bot.action.LegacyDevice_.focusOnElement(body)

## Steps to reproduce -

Create an HTML file with the following contents:
```
<input id='t' type='text' value='123'></input>
<script>
count = 0;
document.getElementById('t').addEventListener('blur', () => ++count, true);
</script>
```

Then run the following Python code with ChromeDriver 2.43 (which uses the JavaScript atom mentioned above to implement element clear command):
```
from selenium import webdriver

driver = webdriver.Chrome()
driver.get(URL-to-the-above-HTML-file)
text = driver.find_element_by_id('t')
text.click()
text.clear()

print('count=' + str(driver.execute_script('return count')))
driver.quit()
```

The above code prints "count=2", showing that the blue event handler is called twice.

**Comment by lmtierney** [comment]
They do use the atom from selenium, but I'm not sure updating it here means they will update it on their end (https://chromium.googlesource.com/chromium/src/+/master/third_party/webdriver/atoms.cc)

**Comment by lmtierney** [comment]
You may want to raise a bug with them referring to this one as well

**Comment by JohnChen0** [comment]
Created an issue for ChromeDriver at https://crbug.com/chromedriver/2622. I work for ChromeDriver, and can update the atoms in ChromeDriver if this issue is fixed from Selenium side. If there is an agreement that the right fix is to remove the explicit call to raise blue event from the atom, I'd be happy to submit a PR for it. (An alternative would be patching the atom directly inside ChromeDriver, but I'd rather avoid that if possible.)

**Comment by lmtierney** [comment]
Ah, perfect then! @barancev, thoughts?

**Comment by JohnChen0** [comment]
Resolved with PR #7102.
